docs/tutorials/PluginDevelopmentGuide.md

```diff
@@ -1580,11 +1601,29 @@ PLUGIN_EXPORT void PLUGIN_CALL ProcessTick()
     }
 }
 ```
+## Error C2371 int32_t redefinition diferent types
+This error occurs without this #define HAVE_STDINT_H, at the top of your project
+just add
+
+```cpp
+
+#define HAVE_STDINT_H
```

Comment:
This shouldn't be done in code but in the project's preprocessor settings; also, there are 3 other defines that can possibly be set - HAVE_UNISTD_H, HAVE_INTTYPES_H, HAVE_ALLOCA_H